docs/tempo/website/grafana-agent/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Grafana Agent
+weight: 400
+---
+
+# Grafana Agent
+
+The [Grafana Agent](https://github.com/grafana/agent) is a telemetry 
+collector for sending metrics, logs, and trace data to the opinionated 
+Grafana observability stack.
+
+It is commonly used as a tracing pipeline, offloading traces from the 
+application and forwarding them to a storage backend.
+The Grafana Agent tracing stack is built using OpenTelemetry.
+
+The Grafana Agent supports receiving traces in multiple formats:
+OTLP (OpenTelemetry), Jaeger, Zipkin and OpenCensus.
+
+On top of receiving and exporting traces, the Grafana Agent contains many 
+features that make your distributed tracing system more robust, and 
+leverages all the data that is processed in the pipeline.
+
+## Architecture
+
+The Grafana Agent can run a set of tracing instances, each of which configures its own pipeline.
+Each pipeline is built using OpenTelemetry,
+and its architecture mirrors that of the OTel Collector's [design](https://github.com/open-telemetry/opentelemetry-collector/blob/846b971758c92b833a9efaf742ec5b3e2fbd0c89/docs/design.md).
+See the [configuration reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config) for all available config options. 
+For a quick start, refer to this [blog post](https://grafana.com/blog/2020/11/17/tracing-with-the-grafana-agent-and-grafana-tempo/).
+
+<p align="center"><img src="https://raw.githubusercontent.com/open-telemetry/opentelemetry-collector/846b971758c92b833a9efaf742ec5b3e2fbd0c89/docs/images/design-pipelines.png" alt="Tracing pipeline architecture"></p>
+
+Having multiple tracing pipelines allows you to configure multiple distinct 
+pipelines, each of which collects separate spans and sends them to different 
+backends.
+
+### Receiving traces
+
+The Grafana Agent supports multiple ingestion methods: 
+OTLP (OpenTelemetry), Jaeger, Zipkin, OpenCensus and Kafka.
+
+Each tracing pipelines can be configured to receive traces in all these formats.
+Traces that arrive to a pipeline will enter the same tracing 
+pipeline, and will be processed equally.
```

Comment:
This sounds a bit confusing.

Reply:
Removed that last line, it should be clearer now